src/test/javascript/spec/component/assessment-shared/assessment-header.component.spec.ts

```diff
@@ -25,6 +30,9 @@ describe('AssessmentHeaderComponent', () => {
                     provide: JhiAlertService,
                     useClass: JhiAlertService, // use the real one
                 },
+                { provide: SessionStorageService, useClass: MockSyncStorage },
```

Comment:
As currently discussed on slack, using MockProvider might be prefferrable

Reply:
```
{ provide: SessionStorageService, useClass: MockSyncStorage },
{ provide: LocalStorageService, useClass: MockSyncStorage },
```
these two dependencies are needed as they were braking the initial PR tests which had to then be reverted. both these dependencies are used extensively as imports all around the test directory. There are about ~100 test files including them as a provider. We thought about finding ways to remove this double liner altogether, and re-use it instead but on a separate PR.